src/organisation/components/pba-numbers-form/pba-numbers-form.component.spec.ts

```diff
@@ -6,20 +6,17 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { ExuiCommonLibModule } from '@hmcts/rpx-xui-common-lib';
 import { Action, Store } from '@ngrx/store';
 import { RxReactiveFormsModule } from '@rxweb/reactive-form-validators';
-import { of } from 'rxjs';
-
+import { of } from 'rxjs/internal/observable/of';
```

Comment:
I don't think this should be deep nested - suggest reverting back to import { of } from 'rxjs';

Reply:
Its now shallow nested